environments/icds-cas/app-processes.yml

```diff
@@ -573,7 +577,11 @@ pillows:
       start_process: 9
       num_processes: 9
       total_processes: 120
-  'pillow7':
+    case_messaging_sync_pillow:
```

Comment:
These `case_messaging_sync_pillow` changes are only around until the backlog gets processed. I'll likely revert before EOD and deploy

Reply:
reverted and updated supervisor confs